### PR TITLE
Updated Dockerfiles

### PR DESCRIPTION
**Pull request by Jaff**
Poor practice last time, did not perform `docker build ...` and test resulting containers. 
Proper steps this time, built, run, performed test builds against UDF code.

### CODE REVIEW

**Review by llamahunter — COMMENTED** [review]
lgtm, but waiting for @arohter to chime in.

File: 11/jdk/alpine/Dockerfile.hotspot.releases.full.maven [hunk 1]
```diff
@@ -0,0 +1,21 @@
+
+FROM docker.tivo.com/openjdk11:alpine
+
+ARG MAVEN_VERSION=3.6.1
+ARG USER_HOME_DIR="/root"
+ARG SHA=b4880fb7a3d81edd190a029440cdf17f308621af68475a4fe976296e71ff4a4b546dd6d8a58aaafba334d309cc11e638c52808a4b0e818fc0fd544226d952544
```
Comment (arohter) [inline 11/jdk/alpine/Dockerfile.hotspot.releases.full.maven]: What is `SHA` for ?  Not used afaik.

File: 11/jdk/alpine/Dockerfile.hotspot.releases.full.maven [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+ARG BASE_URL=https://apache.osuosl.org/maven/maven-3/${MAVEN_VERSION}/binaries
+
+RUN apk add --update --no-cache wget curl bash && \
+    mkdir -p /usr/share/maven && \
+    curl -fsSL -o /tmp/apache-maven.tar.gz ${BASE_URL}/apache-maven-${MAVEN_VERSION}-bin.tar.gz && \
+    tar -xzf /tmp/apache-maven.tar.gz -C /usr/share/maven --strip-components=1 && \
+    rm -f /tmp/apache-maven.tar.gz && \
+    ln -s /usr/share/maven/bin/mvn /usr/bin/mvn
+
+ENV JAVA_HOME=/opt/java/openjdk \
```
Comment (arohter) [inline 11/jdk/alpine/Dockerfile.hotspot.releases.full.maven]: Don't overwrite parent image's variables.  `JAVA_HOME`

File: 11/jdk/alpine/Dockerfile.hotspot.releases.full.maven [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    MAVEN_HOME=/usr/share/maven \
+    MAVEN_CONFIG="$USER_HOME_DIR/.m2" \
```
Comment (arohter) [inline 11/jdk/alpine/Dockerfile.hotspot.releases.full.maven]: Don't hardcode/re-define/dupe home, re-use the already-declared var `$HOME`:

```
/ # export
export HOME='/root'
export HOSTNAME='626c441363b3'
export JAVA_HOME='/opt/java/openjdk'
export JAVA_VERSION='jdk-11.0.4+11'
export LANG='en_US.UTF-8'
export LANGUAGE='en_US:en'
export LC_ALL='en_US.UTF-8'
export PATH='/opt/java/openjdk/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin'
export PWD='/'
export SHLVL='1'
export TERM='xterm'

```

**Review by arohter — CHANGES_REQUESTED** [review]
See inline comments.

File: 11/jdk/alpine/Dockerfile.hotspot.releases.full.maven (hunk 1 above)
Comment (Jaff) [reply 11/jdk/alpine/Dockerfile.hotspot.releases.full.maven]: Copy/paste from somewhere

File: 11/jdk/alpine/Dockerfile.hotspot.releases.full.maven (hunk 2 above)
Comment (Jaff) [reply 11/jdk/alpine/Dockerfile.hotspot.releases.full.maven]: Fixed now

File: 11/jdk/alpine/Dockerfile.hotspot.releases.full.maven (hunk 3 above)
Comment (Jaff) [reply 11/jdk/alpine/Dockerfile.hotspot.releases.full.maven]: Fixed now

**Comment by arohter** [comment]
👍

**Comment by arohter** [comment]
https://builds-inception.tivo.com/job/adoptopenjdk/33/

**Pull request merged**